hoogle-test/ref/Bug873/test.txt

```diff
@@ -0,0 +1,26 @@
+-- Hoogle documentation, generated by Haddock
+-- See Hoogle, http://www.haskell.org/hoogle/
+
+@package test
+@version 0.0.0
+
+module Bug873
+
+-- | Application operator. This operator is redundant, since ordinary
+--   application <tt>(f x)</tt> means the same as <tt>(f <a>$</a> x)</tt>.
+--   However, <a>$</a> has low, right-associative binding precedence, so it
+--   sometimes allows parentheses to be omitted; for example:
+--   
+--   <pre>
+--   f $ g $ h x  =  f (g (h x))
+--   </pre>
+--   
+--   It is also useful in higher-order situations, such as <tt><a>map</a>
+--   (<a>$</a> 0) xs</tt>, or <tt><a>zipWith</a> (<a>$</a>) fs xs</tt>.
+--   
+--   Note that <tt>($)</tt> is levity-polymorphic in its result type, so
+--   that foo $ True where foo :: Bool -&gt; Int# is well-typed
+($) :: () => (a -> b) -> a -> b
```

Comment:
Darn, I was faintly hoping that fixing this issue would also fix this strange empty context that appears here. Oh well, that can wait for another day.

Reply:
I was hoping for this too, but I didn't see any clean way to abstract that logic across backends too.

Reply:
That makes sense. It might be worth opening a separate issue about this, then.